torch/jit/__init__.py

```diff
@@ -656,6 +656,25 @@ def __getitem__(self, k):
             raise KeyError(k)
         return self.module._get_parameter(k)
 
+# types that can be constants. If you edit this list,
+# then you also need to edit the handlers in ConstantValue in jit/script/init.cpp
+_constant_types = [bool, float, int, Module, types.FunctionType]
+
+
+def _get_valid_constant(v):
+    if any(isinstance(v, typ) for typ in _constant_types):
+        return v
+    elif isinstance(v, tuple) or isinstance(v, list):
+        return tuple(_get_valid_constant(x) for x in v)
+    constants = ", ".join(typ.__name__ for typ in _constant_types)
+    raise TypeError(
+        "'{}' object is not a valid constant or a list/tuple of valid constants. ".format(type(v).__name__) +
+        "Valid constants are {{{}}}".format(constants))
+
+
+class Const(object):
```

Comment:
I know this might be bikeshedding, but can we please make a function for creating constants and call it `const` (and rename this to `_Const` or sth like that). I think it would be more consistent with other API we expose, and wouldn't mean "create this class", but only "mark this as const" (somehow, doesn't matter how).